docs/references/experimental/monorepo/README.md

```diff
@@ -0,0 +1,3 @@
+## Monorepo Projects
+
+TODO: Work with Enrico to fill this out.
```

Comment:
@enricozb would you be willing to pair on this (or alternately, feel free to just provide your own doc in a similar style to the others I've added)?

Reply:
Down to pair